src/openforms/js/components/form/select.js

```diff
@@ -1,5 +1,175 @@
-import {Formio} from 'formiojs';
-import {defineChoicesEditFormTabs} from './abstract';
+import {Formio, Utils} from 'formiojs';
+import DEFAULT_TABS, {ADVANCED, BASIC, REGISTRATION, VALIDATION} from './edit/tabs';
 
+const Select = Formio.Components.components.select;
 
-defineChoicesEditFormTabs(Formio.Components.components.select, 'data.values');
+const values = [{
+    type: 'datagrid',
+    input: true,
+    label: 'Values',
+    key: 'data.values',
+    tooltip: 'The radio button values that can be picked for this field. Values are text submitted with the form data. Labels are text that appears next to the radio buttons on the form.',
+    weight: 10,
+    reorder: true,
+    defaultValue: [{label: '', value: ''}],
+    components: [
+        {
+            label: 'Label',
+            key: 'label',
+            input: true,
+            type: 'textfield',
+        },
+        {
+            label: 'Value',
+            key: 'value',
+            input: true,
+            type: 'textfield',
+            allowCalculateOverride: true,
+            calculateValue: {_camelCase: [{var: 'row.label'}]},
+        },
+    ],
+}];
+
+
+const APPOINTMENT = {
+    key: 'appointment',
+    label: 'Appointment',
+    components: [
+        {
+            type: 'checkbox',
+            key: 'showProducts',
```

Comment:
I'd like to see all the appointment configuration namespaced in a nested object to prevent (future) key collisions

```suggestion
            key: 'appointments.showProducts',
```
Of course this applies for the other components too below this one :-)